.github/workflows/test.yml

```diff
@@ -1,262 +1,48 @@
-name: Tests / Code Coverage
-#  Tests / Code Coverage workflow runs unit tests and uploads a code coverage
-#  report
-#  This workflow is run on pushes to main & every Pull Requests where a .go,
-#  .mod, .sum have been changed
+name: test
```

Comment:
```suggestion
name: Tests / Code Coverage
```

Because, if I understand right, this workflow is still sending coverage report

Reply:
it is sending a coverage report but the rename is motivated by https://github.com/celestiaorg/celestia-app/issues/1796

the workflow name doesn't need to include the words "code coverage" because the job name will include it. For example:

<img width="514" alt="Screenshot 2023-05-22 at 8 40 46 AM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/f6c933d6-eb1c-4eb1-850a-2e4a761d9e54">